### PR TITLE
Resolve issues Nov 23

### PR DESCRIPTION
**Pull request by mattwojo**

### CODE REVIEW

File: WSL/wsl-config.md [hunk 1]
```diff
@@ -15,43 +15,34 @@ The [wsl.conf](#wslconf) and [.wslconfig](#wslconfig) files are used to configur
...
+
+## The 8 second rule for configuration changes
+
+You must wait until the subsystem running your Linux distribution completely stops running and restarts for configuration setting updates to appear. This typically takes about 8 seconds after closing ALL instances of the distribution shell.
+
+If you launch a distribution (e.g. Ubuntu), modify the configuration file, close the distribution, and then re-launch it, you might assume that your configuration changes have immediately gone into effect. This is not currently the case as the subsystem could still be running. You must wait for the subsystem to stop before relaunching in order to give enough time for your changes to be picked up. You can check to see whether your Linux distribution (shell) is still running after closing it by using PowerShell with the command: `wsl --list --running`. If no distributions are running, you will receive the response: "There are no running distributions." You can now restart the distribution to see your configuration updates applied.
```
Comment (craigloewen-msft) [inline WSL/wsl-config.md:29]: You can also use `wsl --terminate <distroName>` to terminate a running distro instantly.

**Review by craigloewen-msft — APPROVED** [review]

File: WSL/wsl-config.md (hunk 1 above)
Comment (mattwojo) [reply WSL/wsl-config.md:29]: Added a note in the doc about this - thanks!

File: WSL/networking.md [hunk 2]
```diff
@@ -57,6 +57,16 @@ netsh interface portproxy add v4tov4 listenport=4000 listenaddress=0.0.0.0 conne
...
 
+- `wsl hostname -I` for the IP address of your Linux distribution installed via WSL 2 (the WSL 2 VM address)
+- `cat /etc/resolv.conf` for the IP address of the Windows machine as seen from WSL 2 (the WSL 2 VM)
+
+Using `listenaddress=0.0.0.0` will listen on all [IPv4 ports](https://stackoverflow.com/questions/9987409/want-to-know-what-is-ipv4-and-ipv6#:~:text=The%20basic%20difference%20is%20the,whereas%20IPv6%20has%20128%20bits.).
+
+> [!NOTE]
+> Using a lowercase "i" with the hostname command will generate a different result than using an uppercase "I". `wsl hostname -i` is your local machine (127.0.1.1 is a placeholder diagnostic address), whereas `wsl hostname -I` will return your local machine's IP address as seen by other machines and should be used to identify the `connectaddress` of your Linux distribution running via WSL 2.
+
+## IPv6 access
```
Comment (taoyouh) [inline WSL/networking.md:68]: Hi, I don't quite understand what this "IPv6 access" session is for. The things below looks irrelevant to IPv6 access.
